rama-net/src/tls/server/config.rs

```diff
@@ -92,6 +96,12 @@ impl Default for ServerCertIssuerKind {
     }
 }
 
+impl ServerCertIssuerKind {
+    pub fn from_dynamic_issuer<T: DynamicCertIssuer + Send + Sync + 'static>(issuer: T) -> Self {
```

Comment:
Probably best to instead just set the SS'S trait bounds enforced on the DynamicCertIssuer already, similar to Service.

I imagine we can globally be more clever about this in a year or two, but for now this is what it is